content/en/blog/_posts/2023-05-15-kubernetes-1.27-updates-on-speeding-up-pod-startup.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: "Kubernetes 1.27: updates on speeding up Pod startup"
```

Comment:
Better avoid dots in the file name, replace `1.27` with `1-27`.

Reply:
Updated. Thanks.